requirements.txt

```diff
@@ -1,3 +1,3 @@
-covalent==0.177.0.post1.dev1
+covalent==0.220.0.post2
```

Comment:
Let's remove the exact pins from this file and have `>=` instead. Also, the minimum version of covalent should now be `0.232.0` instead. TIA 🙏 !